addons/website_slides/static/src/js/slides_share.js

```diff
@@ -29,9 +36,10 @@ var ShareMail = publicWidget.Widget.extend({
                     email: input.val(),
                 },
             }).then(function () {
-                self.$el.html($('<div class="alert alert-info" role="alert">' + _t('<strong>Thank you!</strong> Mail has been sent.') + '</div>'));
+                self.$el.html($('<div class="alert alert-info" role="alert">' + _t('<strong>Sharing is caring!</strong> Email(s) sent.') + '</div>'));
```

Comment:
Isn't it possible to put this message in the dom with a class `d-none` and remove the class to show the error ?

So you will not need to call `.html` and it will be easier to see where the message is inserted, what do you think ?